apps/helpers/helpers.py

```diff
@@ -5,9 +5,9 @@
 from django.conf import settings
 from django.template import engines
 
-from ..models import AppInstance, AppStatus  # type: ignore
-from ..serialize import serialize_app  # type: ignore
-from ..tasks import deploy_resource  # type: ignore
+from apps.models import AppInstance, AppStatus
+from apps.serialize import serialize_app
+from apps.tasks import deploy_resource
```

Comment:
Great! This has bothered me

Reply:
Are you against relative imports? :)

Reply:
Hehe well i think so. I like more explicit stuff. But I can change my mind.

Reply:
This also fixes mypy error: Relative import climbs too many namespaces.